lib/matplotlib/__init__.py

```diff
@@ -1318,11 +1318,11 @@ def use(arg, warn=True, force=False):
         If True, warn if this is called after pyplot has been imported
         and a backend is set up.
 
-        defaults to True
+        defaults to False
 
     force : bool, optional
         If True, attempt to switch the backend.  This defaults to
-        False.
+        True.
```

Comment:
Mention that this will raise ImportError on failure, which occurs if an incompatible event loop has already started.